src/function/cast/bit_cast.cpp

```diff
@@ -10,7 +10,7 @@ BoundCastInfo DefaultCasts::BitCastSwitch(BindCastInput &input, const LogicalTyp
 		// bit to varchar
 		return BoundCastInfo(&VectorCastHelpers::StringCast<string_t, duckdb::CastFromBit>);
 	case LogicalTypeId::BLOB:
-		return DefaultCasts::ReinterpretCast;
+		return BoundCastInfo(&VectorCastHelpers::StringCast<string_t, duckdb::CastBitToBlob>);
```

Comment:
Should we support bit <-> blob casts at all? It seems that Postgres does not support them:

```sql
postgres=# select cast('01010101' as bit(8))::bytea;
ERROR:  cannot cast type bit to bytea
LINE 1: select cast('01010101' as bit(8))::bytea;
                                         ^
postgres=# SELECT cast(bytea '\xDEADBEEF' as bit(100));
ERROR:  cannot cast type bytea to bit
LINE 1: SELECT cast(bytea '\xDEADBEEF' as bit(100));

```

Reply:
I had initially added it to accommodate adding bit type to test_all_types for parquet, but indeed there seems no real reason to support this, so I've removed it altogether!